packages/patternlab/styleguide/source/assets/js/modules/pageFlipper.js

```diff
@@ -0,0 +1,34 @@
+// let arrowNavItems = document.querySelectorAll(".ma__arrow-nav");
+let arrowButtons = document.querySelectorAll(".ma__arrow-button");
+
+function switchVisibility() {
+  let width = document.querySelector("body").clientWidth;
+  // Make arrow buttons visible to keyboard and screen readers.
+  if (width < 621) {
+    arrowButtons.forEach(button => {
+      if(button.hasAttribute("aria-hidden")) {
+        button.removeAttribute("aria-hidden");
+      }
+
+      if (button.getAttribute("tabindex") == "-1") {
+        button.setAttribute("tabindex", "0");
+      }
+    });
+  }
+
+  // Hide arrow buttons to keyboard and screen readers.
+  if (width > 620) {
+    arrowButtons.forEach(button => {
+      if(!button.hasAttribute("aria-hidden")) {
+        button.setAttribute("aria-hidden", "true");
+      }
+
+      if (button.getAttribute("tabindex") == "0") {
```

Comment:
I don't think the check works, might be because the `tabindex` doesn't exist? Can we chat about your intention for this JS to work? 
https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-22292_binder-page-a11y-improvements/?p=organisms-page-flipper

Reply:
This JS currently applies to all arrow buttons, is it too greedy?  If ArrowButton is only used in PageFlippers, and we want to hide the button links, should we just in PageFlippers set ariaHidden to true? Then under all screen sizes it will hide the duplicated links and we don't need this JS?

Reply:
The selector is narrowed down as suggested.